.eslintrc.json

```diff
@@ -27,7 +27,8 @@
       {
         "argsIgnorePattern": "^_",
         "varsIgnorePattern": "^_",
-        "caughtErrorsIgnorePattern": "^_"
+        "caughtErrorsIgnorePattern": "^_",
+        "ignoreRestSiblings":true
```

Comment:
nice this is great

Reply:
theres one place in the oo sdk where we strip out properties by using an object destructure. without this flag, linting will pick it up as unused variables, which is really annoying.